source/logic/plugins/PluginLogic.py

```diff
@@ -23,9 +23,10 @@ def get_required_info(self, organization, plugin_name):
         try:
             required_info = self._plugin_manager.get_required_info_of(
                 organization, plugin_name)
-            required_info["organization"] = organization
-            required_info["plugin_name"] = plugin_name
-            required_info["name"] = "default"
-            return required_info
+            required_info['name'] = 'default_name'
```

Comment:
Would be nice if we could eventually let the plugin developer decide on what the default name will be.